src/main/java/org/example/homework8/Bedroom/Bed.java

```diff
@@ -0,0 +1,36 @@
+package org.example.homework8.Bedroom;
+
+public class Bed {
+    private String style;
+    private int pillows;
+    private int height;
+    private int sheets;
+    private int quilt;
+    public Bed(String style, int pillows, int height, int sheets, int quilt) {
+        this.style = style;
+        this.pillows = pillows;
+        this.height = height;
+        this.sheets = sheets;
+        this.quilt = quilt;
+    }
+
+    public static void make(){
+        System.out.println("The bed is being made");
+    }
+
+    public String getStyle(){
+        return this.style;
```

Comment:
```suggestion
        return style;
```
using this in getters is not mandatory. i guess it looks better without it here

Reply:
Improved.